src/packages/@app/pages/EnvironmentPage/EnvironmentPage.ViewModel.ts

```diff
@@ -316,3 +331,10 @@ export class EnvironmentViewModel {
     }
   };
 }
+function onFetchEnvironmentGuide(arg0: { id: string }) {
+  console.error("Function not implemented.");
+}
+
+function onFetchCollectionGuide(arg0: { id: string }) {
+  console.error("Function not implemented.");
+}
```

Comment:
this complete view model is wrapped in a class, you only need to define function in the class scope only.